ql/src/test/queries/clientpositive/sysdb.q

```diff
@@ -1,5 +1,6 @@
 --! qt:dataset:src,part,srcbucket:ONLY
 --! qt:sysdb
+--! qt:replace:/(initiated\s+NULL\s+NULL\s+)[0-9]*(\s+NULL)/$1#Masked#$2/
```

Comment:
Nit: maybe add a comment here about what's going on, to save someone time down the road

Reply:
Done